src/assets/stylesheets/theme/loader.scss

```diff
@@ -38,6 +38,7 @@
   );
   -webkit-animation: load3 1.4s infinite linear;
   animation: load3 1.4s infinite linear;
+  z-index: 1;
```

Comment:
FYI: The loader is definitely an improvement. But you're still able to interact with the other "cart" elements, while the cart is submitting (e.g. removing items from the cart). I don't have a good idea how to improve this, since those widgets are independent from the `CheckoutButtonWidget`.

Reply:
I think state of the art would be showing the spinner in a `<dialog>` wrapper opened with `showModal()`.

Reply:
Sounds interesting. Could you check, if this is easily buildable?

Reply:
#495